packages/fx-core/resource/package.nls.json

```diff
@@ -375,35 +375,35 @@
-  "core.createProjectQuestion.officeXMLAddin.excel.react.title": "Add-in with React framework",
+  "core.createProjectQuestion.officeXMLAddin.excel.react.title": "Add-in with React Framework",
   "core.createProjectQuestion.officeXMLAddin.excel.react.detail": "Create an Excel add-in with React framework",
-  "core.createProjectQuestion.officeXMLAddin.excel.cf.shared.title": "Excel Custom Functions using a Shared Runtime",
+  "core.createProjectQuestion.officeXMLAddin.excel.cf.shared.title": "Excel Custom Functions Using a Shared Runtime",
   "core.createProjectQuestion.officeXMLAddin.excel.cf.shared.detail": "Create an Excel add-in leveraging Custom Functions using a Shared Runtime",
-  "core.createProjectQuestion.officeXMLAddin.excel.cf.js.title": "Excel Custom Functions using a JavaScript-only Runtime",
+  "core.createProjectQuestion.officeXMLAddin.excel.cf.js.title": "Excel Custom Functions Using a JavaScript-only Runtime",
   "core.createProjectQuestion.officeXMLAddin.excel.cf.js.detail": "Create an Excel add-in leveraging Custom Functions using a JavaScript-only Runtime",
   "core.createProjectQuestion.officeXMLAddin.excel.create.title": "Create an Excel Add-in",
   "core.createProjectQuestion.officeXMLAddin.powerpoint.title": "PowerPoint Add-in",
   "core.createProjectQuestion.officeXMLAddin.powerpoint.detail": "Build engaging solutions for presentations across platform",
-  "core.createProjectQuestion.officeXMLAddin.powerpoint.sso.title": "Create a PowerPoint add-in with Single Sign On capabilities",
+  "core.createProjectQuestion.officeXMLAddin.powerpoint.sso.title": "Add-in with Single Sign On",
   "core.createProjectQuestion.officeXMLAddin.powerpoint.sso.detail": "PowerPoint add-in with Single Sign On capabilities",
-  "core.createProjectQuestion.officeXMLAddin.powerpoint.react.title": "Add-in with React framework",
+  "core.createProjectQuestion.officeXMLAddin.powerpoint.react.title": "Add-in with React Framework",
   "core.createProjectQuestion.officeXMLAddin.powerpoint.react.detail": "Create a PowerPoint add-in with React framework",
   "core.createProjectQuestion.officeXMLAddin.powerpoint.create.title": "Create a PowerPoint Add-in",
-  "core.createProjectQuestion.officeXMLAddin.taskpane.title": "Add-in with Basic Taskpane",
-  "core.createProjectQuestion.officeXMLAddin.taskpane.detail": "Customize the Ribbon with a button and embed content in the Taskpane",
-  "core.createProjectQuestion.officeXMLAddin.manifestOnly.title": "Add-in project containing the manifest only",
+  "core.createProjectQuestion.officeXMLAddin.taskpane.title": "Add-in with Basic Task Pane",
+  "core.createProjectQuestion.officeXMLAddin.taskpane.detail": "Customize the Ribbon with a button and create a dashboard in the Task Pane",
+  "core.createProjectQuestion.officeXMLAddin.manifestOnly.title": "Add-in Project Containing the Manifest Only",
```

Comment:
Proposed:
Add-in Project With only Manifest File
Logic: To be consistent with the description and also to follow the usual adjective-noun order in English and sounds more natural.